test/runtime/pod.go

```diff
@@ -107,6 +146,28 @@ func (builder *PodBuilder) AddConfigMapVolume(name, configMapName string) *PodBu
 	return builder
 }
 
+func (builder *PodBuilder) AddEmptyDirVolume(name string) *PodBuilder {
```

Comment:
suggest to refactor the three functions into one. `AddVolumeFromSource(name, VolumeSource)`